realm/lib/realm-chat.hoon

```diff
@@ -250,26 +250,18 @@
   =/  note=push-notif
     [
       app-id=app-id.state
-      data=[path-row unread avatar message]
+      data=[path-row avatar message]
       title=(malt ~[['en' title]])
       subtitle=?:(=(subtitle '') ~ (malt ~[['en' subtitle]]))
       contents=(malt ~[['en' content]])
     ]
   ::  send http request
   ::
-  =/  =header-list:http    ['Content-Type' 'application/json']~
-  =|  =request:http
-  :: TODO include the unread count in the push notif (perhaps global?)
-  =:  method.request       %'POST'
-      url.request          'https://onesignal.com/api/v1/notifications'
-      header-list.request  header-list
-      body.request
-        :-  ~
-        %-  as-octt:mimes:html
-        %-  en-json:html
-        %+  notify-request:encode
-          note
-        devices.state
+  =/  =request:http
+  :*  method=%'POST'
+      url='https://onesignal.com/api/v1/notifications'
+      header-list=['Content-Type' 'application/json']~
+      body=(as-octt:mimes:html (en-json:html (notify-request:encode note devices.state unread)))
```

Comment:
Think I need a nudge here. I'm getting the type error:

```hoon
/lib/realm-chat/hoon::[261 3].[265 5]>
mint-nice
-need.u([p=@ud q=@])
-have.[p=@ud q=@]
nest-fail
```

Even though unread is supposedly `@ud` and that's what the `notify-request` signature expects.

Reply:
well you changed the body from a cell of `[~ as-octt]` to just `as-octt` so that's probably a problem